scripts/release-util.ts

```diff
@@ -146,7 +149,8 @@ export const E2E_PHASE: Phase = {
   packages: ['e2e'],
   deps: [
     'tfjs', 'tfjs-backend-cpu', 'tfjs-backend-wasm', 'tfjs-backend-webgl',
-    'tfjs-converter', 'tfjs-core', 'tfjs-data', 'tfjs-layers', 'tfjs-node'
+    'tfjs-backend-webgpu', 'tfjs-converter', 'tfjs-core', 'tfjs-data',
+    'tfjs-layers', 'tfjs-node'
```

Comment:
Add tfjs webgpu to the e2e packages so it gets correctly rewritten in verdaccio tests.